src/inventory/CallbackInventoryChangeListener.php

```diff
@@ -75,6 +75,10 @@ public function onSlotChange(Inventory $inventory, int $slot, Item $oldItem) : v
 		($this->onSlotChangeCallback)($inventory, $slot, $oldItem);
 	}
 
+	/**
+	 * @param Inventory $inventory
```

Comment:
the @param tag is only needed on iterable types, this line isn't needed

Reply:
more accurately, @param and @return are now only required when the native typehint doesn't fully describe the type.